.circleci/config.yml

```diff
@@ -284,3 +290,23 @@ jobs:
           command: |
             sudo npm install -g markdown-link-check
             bash .circleci/check-links/check-links.sh
+
+  windows-msi:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - attach_to_workspace
+      - run:
+          name: Install Wix Toolset
+          command: |
+            choco install wixtoolset
+            setx /m PATH "%PATH%;C:\Program Files (x86)\WiX Toolset v3.11\bin"
```

Comment:
Annoyingly we have to do this because the choco package doesn't take care of it for us: https://chocolatey.org/packages/wixtoolset#comment-3471097677